From 5a4a6664b474af52df429daf3b81d9cd5a98d911 Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Mon, 18 Mar 2024 11:08:07 +0100
Subject: [PATCH 1073/1141] arm64: dts: st: rename IOM pins muxing
 configurations for stm32mp25

Rename IOM pins muxing configurations to distinguish
the OSPI instances and the OSPI ports.

Change-Id: I6045ca66bef7d6808a9f845465b4ed31b162f60c
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/368088
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrice CHOTARD <patrice.chotard@foss.st.com>
Reviewed-by: Patrice CHOTARD <patrice.chotard@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
---
 arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi | 12 ++++++------
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts    | 14 ++++++--------
 2 files changed, 12 insertions(+), 14 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi b/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi
index 60f07008b3ee..7187aaaf1c55 100644
--- a/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp25-pinctrl.dtsi
@@ -398,7 +398,7 @@ pins {
 		};
 	};
 
-	ospi1_clk_pins_a: ospi1-clk-0 {
+	ospi_port1_clk_pins_a: ospi-port1-clk-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 0, AF10)>; /* OSPI1_CLK */
 			bias-disable;
@@ -407,13 +407,13 @@ pins {
 		};
 	};
 
-	ospi1_clk_sleep_pins_a: ospi1-clk-sleep-0 {
+	ospi_port1_clk_sleep_pins_a: ospi-port1-clk-sleep-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 0, ANALOG)>; /* OSPI1_CLK */
 		};
 	};
 
-	ospi1_cs0_pins_a: ospi1-cs0-0 {
+	ospi_port1_cs0_pins_a: ospi-port1-cs0-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 3, AF10)>; /* OSPI_NCS0 */
 			bias-pull-up;
@@ -422,13 +422,13 @@ pins {
 		};
 	};
 
-	ospi1_cs0_sleep_pins_a: ospi1-cs0-sleep-0 {
+	ospi_port1_cs0_sleep_pins_a: ospi-port1-cs0-sleep-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 3, ANALOG)>; /* OSPI_NCS0 */
 		};
 	};
 
-	ospi1_io03_pins_a: ospi1-io03-0 {
+	ospi_port1_io03_pins_a: ospi-port1-io03-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 4, AF10)>, /* OSPI_IO0 */
 				 <STM32_PINMUX('D', 5, AF10)>, /* OSPI_IO1 */
@@ -440,7 +440,7 @@ pins {
 		};
 	};
 
-	ospi1_io03_sleep_pins_a: ospi1-io03-sleep-0 {
+	ospi_port1_io03_sleep_pins_a: ospi-port1-io03-sleep-0 {
 		pins {
 			pinmux = <STM32_PINMUX('D', 4, ANALOG)>, /* OSPI_IO0 */
 				 <STM32_PINMUX('D', 5, ANALOG)>, /* OSPI_IO1 */
diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 6c01bf40246f..d3426112b5cd 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -536,14 +536,12 @@ &ommanager {
 	memory-region = <&mm_ospi1>;
 	memory-region-names = "mm_ospi1";
 	pinctrl-names = "default", "sleep";
-	pinctrl-0 = <&ospi1_clk_pins_a
-		     &ospi1_io03_pins_a
-		     &ospi1_cs0_pins_a>;
-
-	pinctrl-1 = <&ospi1_clk_sleep_pins_a
-		     &ospi1_io03_sleep_pins_a
-		     &ospi1_cs0_sleep_pins_a>;
-
+	pinctrl-0 = <&ospi_port1_clk_pins_a
+		     &ospi_port1_io03_pins_a
+		     &ospi_port1_cs0_pins_a>;
+	pinctrl-1 = <&ospi_port1_clk_sleep_pins_a
+		     &ospi_port1_io03_sleep_pins_a
+		     &ospi_port1_cs0_sleep_pins_a>;
 	status = "okay";
 
 	spi@40430000 {
-- 
2.39.2

